util/util.go

```diff
@@ -33,3 +33,19 @@ func NewLotusClient(lotusAPI string, lotusToken string) jsonrpc.RPCClient {
 		})
 	}
 }
+
+func ChunkSlice[T any](slice []T, chunkSize int) [][]T {
+	var chunks [][]T
+
+	for i := 0; i < len(slice); i += chunkSize {
+		end := i + chunkSize
+		if end > len(slice) {
+			end = len(slice)
+		}
+		chunks = append(chunks, slice[i:end])
+	}
+
+	return chunks
+}
+
+const BatchSize = 100
```

Comment:
this looks conservatively chosen.
maybe we should file an issue to return to tune it for performance later.

Reply:
Yes, there is a SQLite_MAX_PARAMETER that is 999. We can have different values for different backend